src/Atoms/Trans/useTranslation.js

```diff
@@ -5,8 +5,18 @@ export const { Provider } = Context
 export default () => {
   const translations = useContext(Context) || {}
 
-  return transKey => transKey.split('.').reduce(
-    (acc, k) => acc[k] || transKey,
-    translations
-  )
+  return (transKey, parameters = {}) => {
+    let translated = transKey
+      .split('.')
+      .reduce(
+        (acc, k) => acc[k] || transKey,
+        translations
+      )
+
+    Object.entries(parameters).forEach(([key, value]) => {
+      translated = translated.replace(new RegExp(`%${key}%`, 'g'), value)
+    })
+
+    return translated
+  }
 }
```

Comment:
```js
const getTranslationSelector = translations => transKey => (
  transKey
    .split('.')
    .reduce(
      (acc, k) => acc[k] || transKey,
      translations
    )
)

const replaceParam = (translated, [key, value]) => translated.replace(
  new RegExp(`%${key}%`, 'g'),
  value
)

export default () => {
  const translations = useContext(Context) || {}
  const getTranslation = getTranslationSelector(translations)

  return (transKey, parameters = {}) => (
    Object.entries(parameters)
    .reduce(replaceParam, getTranslation(translations))
  )
}
```

Reply:
what is the point in doing that ?

Reply:
Splitting the code in smaller, simpler functions.
Although I admit that it's not necessary in this case, it's still a good practice to follow the KISS principle and make it a habit.

Reply:
If it's not necessary, I won't do it, feel free to do it in another PR

Reply:
Ok